code/ai/ai_profiles.cpp

```diff
@@ -624,4 +626,22 @@ void ai_profile_t::reset()
     for (int i = 0; i <= MAX_DETAIL_LEVEL; ++i) {
         detail_distance_mult[i] = 0;
     }
+
+	// via Github #2332, enable bugfixes if we are targeting > 19.0
+	if (Targetted_version > gameversion::version(19)) {
```

Comment:
Maybe you could use `mod_supports_version` here. It's pretty much the same except for doing a `>=` so you would use version 20 here to make it use the old behaviour for 19.0.

Reply:
The thing is, that would mean test builds and release candidates wouldn't have the benefit of using those flags.  I could fudge it with 19.0.1, but that seems less elegant.

Reply:
That being said, in this particular instance we moved very quickly to 20.x.  Also, it's `mod_supports_version`, not the *actual* version as used by version-specific commenting.